pkg/cliplugins/bind/plugin/bind_compute.go

```diff
@@ -0,0 +1,295 @@
+	if err != nil {
+		return err
+	}
+
+	desiredAPIExports := sets.NewString(o.APIExports...)
+	existingAPIExports := sets.NewString()
+	for _, binding := range apiBindings.Items {
+		if binding.Spec.Reference.Workspace == nil {
+			continue
+		}
+		existingAPIExports.Insert(fmt.Sprintf("%s:%s", binding.Spec.Reference.Workspace.Path, binding.Spec.Reference.Workspace.ExportName))
+	}
+
+	diff := desiredAPIExports.Difference(existingAPIExports)
+	var errs []error
+	for export := range diff {
+		lclusterName, name := logicalcluster.New(export).Split()
+		apiBinding := &apisv1alpha1.APIBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: apiBindingName(lclusterName, name),
+			},
+			Spec: apisv1alpha1.APIBindingSpec{
+				Reference: apisv1alpha1.ExportReference{
+					Workspace: &apisv1alpha1.WorkspaceExportReference{
+						Path:       lclusterName.String(),
+						ExportName: name,
+					},
+				},
+			},
+		}
+		_, err := client.ApisV1alpha1().APIBindings().Create(ctx, apiBinding, metav1.CreateOptions{})
+		if err != nil && !errors.IsAlreadyExists(err) {
+			errs = append(errs, err)
+		}
+
+		_, err = fmt.Fprintf(o.Out, "apibinding %s for apiexport %s created.\n", apiBinding.Name, export)
+		if err != nil {
+			errs = append(errs, err)
+		}
+	}
+
+	return utilerrors.NewAggregate(errs)
+}
+
+// placement name is a hash of location selectors and ns selector, with location workspace name as the prefix
+func (o *BindWorkloadOptions) placementName() string {
+	clusterName, name := o.ComputeWorkspace.Split()
+	hash := sha256.Sum224([]byte(o.NamespaceSelector + strings.Join(o.LocationSelectors, ",") + clusterName.Path()))
+	base36hash := strings.ToLower(base36.EncodeBytes(hash[:]))
+	return fmt.Sprintf("%s-%s", name, base36hash[:8])
+}
+
+func (o *BindWorkloadOptions) applyPlacement(ctx context.Context, client kcpclient.Interface) error {
+	nsSelector, err := metav1.ParseToLabelSelector(o.NamespaceSelector)
+	if err != nil {
+		return fmt.Errorf("namespace selector format not correct: %w", err)
+	}
+
+	var locationSelectors []metav1.LabelSelector
+	for _, locSelector := range o.LocationSelectors {
+		selector, err := metav1.ParseToLabelSelector(locSelector)
+		if err != nil {
+			return fmt.Errorf("location selector %s format not correct: %w", locSelector, err)
+		}
+		locationSelectors = append(locationSelectors, *selector)
+	}
+
+	placement := &schedulingv1alpha1.Placement{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: o.placementName(),
+		},
+		Spec: schedulingv1alpha1.PlacementSpec{
+			NamespaceSelector: nsSelector,
+			LocationSelectors: locationSelectors,
+			LocationWorkspace: o.ComputeWorkspace.String(),
+			LocationResource: schedulingv1alpha1.GroupVersionResource{
+				Group:    "workload.kcp.dev",
+				Version:  "v1alpha1",
+				Resource: "synctargets",
+			},
+		},
+	}
+
+	_, err = client.SchedulingV1alpha1().Placements().Get(ctx, placement.Name, metav1.GetOptions{})
```

Comment:
Why not handle this the same way as the `APIExport`? Create, return error if one is returned & is not AlreadyExiust